tests/requirements_pytorch.txt

```diff
@@ -0,0 +1,51 @@
+numpy~=2.1.1
+--extra-index-url https://download.pytorch.org/whl/cpu
+torch~=2.4.1; platform_system != "Darwin" or platform_machine != "x86_64"
+torch~=2.2.0; platform_system == "Darwin" and platform_machine == "x86_64"
+
+torchvision~=0.19.1
+transformers~=4.45.1
+pytest~=8.3.3
+defusedxml~=0.7.1
+
+auto-gptq~=0.7.1
+av~=13.0.0
+basicsr~=1.4.2
+datasets~=3.0.1
+easyocr~=1.7.2
+facexlib~=0.3.0
+librosa~=0.10.2
+optimum~=1.22.0
+packaging~=24.1
+pandas~=2.2.3
+protobuf~=5.28.2
+pyctcdecode~=0.5.0
+pytest~=8.3.3
+pytest-html~=4.1.1
+pytest-xdist[psutil]~=3.6.1
+sacremoses~=0.1.1
+sentencepiece~=0.2.0
+soundfile~=0.12.1
+super-image~=0.1.7
+timm~=1.0.9
```

Comment:
Please leave 1.0.8 for now. Model tests in nightly unlikely to pass with 1.0.9, the model list needs to be updated to match the actual list.